config/routes.rb

```diff
@@ -21,5 +21,8 @@
     namespace :v0 do
       resources :ping, only: [:index], constraints: { format: 'json' }
     end
```

Comment:
Yeah Thomas wanted us to do that before we started on the restaurants. So i guess we can delete those tests now.